lib/beaker/dsl/helpers/host_helpers.rb

```diff
@@ -256,13 +256,21 @@ def rsync_to host, from_path, to_path, opts = {}
         #                               'archive/sut-files'.
         #
         # @return [Result] Returns the result of the #scp_from operation.
-        def archive_file_from(host, from_path, opts = {}, archive_root = 'archive/sut-files')
+        def archive_file_from(host, from_path, opts = {}, archive_root = 'archive/sut-files', archive_name = 'sut-files.tgz')
+          require 'archive/tar/minitar'
           filedir = File.dirname(from_path)
           targetdir = File.join(archive_root, host.hostname, filedir)
           FileUtils.mkdir_p(targetdir)
           scp_from(host, from_path, targetdir, opts)
+          create_tarball(archive_root, archive_name)
         end
 
+        def create_tarball(path, archive_name)
```

Comment:
Does this need to be a private method? Maybe not, but, since it is in this host_helper file, it will be injected into the DSL unless you mark it as private. Do you expect users to use the method directly? If so, it should have some yard docs as well.